Form/Examples/ClearForm.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace DUCK.Form.Examples
+{
+	[RequireComponent(typeof(Button))]
+	public sealed class ClearForm : MonoBehaviour
```

Comment:
ClearFormButton?

Reply:
Its not a button, it just requires one.

Reply:
Which pretty makes a button because it has to have a button script to work. I thought a name change would help users understand what this class actually did. By reading ClearForm you expect it to .... Clear a form when actually it listens to a button click to clear the form.

ClearFormButton describes that to me.

Reply:
I didn't want to confuse user by making it seem like the class extends from Button as it isn't actually but rather applies functionality to a button. Is there any other name you can think of that works for both points of view?

Reply:
`FormClearer` haha opening a big can of worms....
what i will say is that `ClearForm` isn't a good name since it's not really describing what it is, names should be nouns. this can be considered a noun if it's a form that is clear. But that's not what this is at all.

I'm wondering if it's really needed. It's an easy way to hook a button click into `form.Reset` you can do that in a few clicks in the editor. Not much more difficult than adding a new component searching for this and putting in in there.

Dunno how you would feel about this (i've not looked through all the code yet) but  different approach be having a SerializedField on the form called `clearButton` and if it was assigned then the form could bind the listener for you